lib/postgres_utility.rb

```diff
@@ -0,0 +1,502 @@
+# frozen_string_literal: true
+
+require_relative "postgres_utility/version"
+
+# rubocop:disable Layout/LineLength, Metrics/ModuleLength
+
+require "English"
+require "active_record"
+
+# PostgresUtility module
+module PostgresUtility
+  extend self
+
+  DEFAULT_BATCH_SIZE = 1000
+
+  # Shorthand for rails environment based connection
+  def rails_connection(cls = nil)
+    cls ||= ActiveRecord::Base
+    cls.connection
+  end
+
+  # Copied from db:migrate:status
+  def pending_migration?
+    db_list = rails_connection
+              .select_values("SELECT version FROM
+                  #{ActiveRecord::Migrator.schema_migrations_table_name}")
+    db_list.map! { |version| format("%.3d", version) }
+    ActiveRecord::Migrator.migrations_paths.each do |path|
+      Dir.foreach(Rails.root.join(path)) do |file|
+        # match "20091231235959_some_name.rb" and "001_some_name.rb" pattern
+        if match_data == /^(\d{3,})_(.+)\.rb$/.match(file)
+          is_up = db_list.delete(match_data[1])
+          return true unless is_up
+        end
+      end
+    end
+    false
+  end
+
+  def db_name
+    rails_connection.current_database
+  end
+
+  def db_connection_config
+    config = ActiveRecord::Base.connection_config
+    config = rails_connection.config if config[:adapter].match(/makara/i)
+    config
+  end
+
+  def db_adapter_name
+    db_connection_config[:adapter]
+  end
+
+  def postgresql?
+    !!db_adapter_name.match(/postgresql/i)
+  end
+
+  def migration_version
+    ActiveRecord::Migrator.current_version
+  end
+
+  def db_version
+    rails_connection.select_value("select version();")
```

Comment:
@sachinsaxena1996 shouldnt we use single quotes here? 
Surprising that rubocop doesnt complain about this?
Can you double check other places in this PR to make sure we are using double quotes only for string interpolation